src/fastoad_cs25/models/handling_qualities/tail_sizing/compute_vt_area.py

```diff
@@ -59,13 +60,18 @@ def compute(self, inputs, outputs, discrete_inputs=None, discrete_outputs=None):
         cruise_mach = inputs["data:TLAR:cruise_mach"]
         # This one is the distance between the 25% MAC points
         wing_htp_distance = inputs["data:geometry:vertical_tail:MAC:at25percent:x:from_wingMAC25"]
+        vtp_area_factor = inputs["settings:geometry:vertical_tail:area_factor"]
```

Comment:
```suggestion
        vtp_area_factor = inputs["tuning:geometry:vertical_tail:area_factor"]
```

Reply:
Corrected in [4b80055](https://github.com/fast-aircraft-design/FAST-OAD_CS25/pull/49/commits/4b800553e8258931efb928e82aa882275b69fe91)